### PR TITLE
Adding typespec base package reference for liftr

### PR DESCRIPTION
**Pull request by nikhgup**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by mikeharder — CHANGES_REQUESTED** [review]
Before this is merged, I have some questions:

1. Where is the source code for `@azure-tools/typespec-liftr-base`?
2. How will `@azure-tools/typespec-liftr-base` version?  It doesn't appear to be using the same version as the other `@azure-tools` or `@typespec` packages.
3. Which team is responsible for updating the version of `@azure-tools/typespec-liftr-base` in main?  Azure SDK or Liftr?
4. Which team is responsible for any issues related to this package?  For example, say the package is updated to a new version, which has breaking changes specs must react to.

**Comment by nikhgup** [comment]
@mikeharder PFB the response 
1. Source code is part of Liftr ADO repo https://dev.azure.com/msazure/Liftr/_git/LiftrBaseApiSpec?path=/packages/typespec-liftr-base/lib/main.tsp
2. Versioning is not followed as per other typespec packages. as This will only be used by LIftr team
3. Version will be updated by Liftr team.
4. Liftr will take care of breaking changes.

This package contains some standard typespec models used in Liftr related RPs. Instead of our teams adding those models in different RP's typespec, we are using this package. for e.g. you can see currently these models are copied as part of each rp https://github.com/Azure/azure-rest-api-specs/tree/main/specification/liftrastronomer

**Comment by mikeharder** [comment]
> Source code is part of Liftr ADO repo https://dev.azure.com/msazure/Liftr/_git/LiftrBaseApiSpec?path=/packages/typespec-liftr-base/lib/main.tsp

Can some members of the Azure SDK team be given read access to this repo?  When I try to browse to https://dev.azure.com/msazure/Liftr I get a 404 (which I assume means I don't have permissions).
<br>
> Version will be updated by Liftr team.  Liftr will take care of breaking changes.

I encourage you to proactively work with the TypeSpec team, to coordinate with their montly releases.  For example, TypeSpec might make a breaking change, which you will need to react to in `typespec-liftr-base`, and publish a new version very shortly after TypeSpec publishes.  You will also likely want to run all your tests against both the current version and the "next" version of TypeSpec, to detect breaking changes that impact you as soon as possible.

To summarize, the Liftr team is responsible for reacting to two types of breaking changes:

1. TypeSpec makes a breaking change, `typespec-liftr-base` needs to react
2. `typespec-liftr-base` makes a breaking change, all specs using `typespec-liftr-base` need to react

For updating specs, the current process is everything in public/main must be updated by the team who made the breaking change to the tool (Azure SDK or Liftr).  The process for RPSaaSMaster is different, where spec authors are expected to update their spec in their next PR.

**Review by mikeharder — APPROVED** [review]

**Comment by nikhgup** [comment]
@mikeharder Yes, we are working with Typespec team and @allenjzhang is aware about the changes. Our typespec files are versioned.
For access, let me check with my team, how can i provide the access to sdk team.

**Comment by allenjzhang** [comment]
@mikeharder, I have went over the maintenance plan with Liftr team. Signing off from my side.

The Liftr team will be pulled in by the EngSys/TypeSpec team if any breaking changes are needed due to TypeSpec releases. The Liftr team needs to make expedited fixes and publish updates promptly.
 
If the Liftr team is making or introducing breaking changes in the Liftr library, these changes will be tested beforehand before upgrading the Liftr package versions in the rest-api-spec repository.

**Review by allenjzhang — APPROVED** [review]
